crates/agglayer-config/src/settlement_service.rs

```diff
@@ -0,0 +1,6 @@
+use serde::{Deserialize, Serialize};
+
+/// The settlement service configuration.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Default)]
+#[serde(rename_all = "kebab-case")]
+pub struct SettlementServiceConfig {}
```

Comment:
Is the settlement service supposed to handle both certificates and validiums? If so, is the config envisioned to be shared between the two or separate?

Reply:
@iljakuklic It will be shared if they use same wallet. As discussed previously we will make per tx config to set those things for validiums/certificates. This configuration will keep the default settings. When needed, every tx submission will have an optional tx configuration to override default values.